tests/acceptance/index-test.ts

```diff
@@ -1,44 +1,38 @@
 import { click, visit } from '@ember/test-helpers';
 import { setupApplicationTest } from 'ember-qunit';
-import Pretender from 'pretender';
 import { module, test } from 'qunit';
+import setupPretender from '../helpers/setup-pretender';
 
 module('Acceptance | index2', hooks => {
   setupApplicationTest(hooks);
-  let server: any;
-  hooks.beforeEach(() => {
-    server = new Pretender();
-  });
-  hooks.afterEach(() => {
-    server.shutdown();
-  });
+  setupPretender(hooks);
 
-  test('visiting /', async assert => {
+  test('visiting /', async function(assert) {
     await visit('/');
     assert.expect(8);
 
-    server.put('/fruits/:id/doRipen', (request: { url: string; requestBody: string }) => {
+    this.server.put('/fruits/:id/doRipen', (request) => {
       const data = JSON.parse(request.requestBody);
       assert.deepEqual(data, { id: '1', name: 'apple' }, 'member action - request payload is correct');
-      assert.equal(request.url, '/fruits/1/doRipen', 'request was made to "doRipen"');
```

Comment:
This `url` property doesn't exist on the type definition for Pretender, and if you think about it, asserting against the entire URL doesn't make _that_ much sense anyway.

We know the structure of the URL is correct because the handler is active. If we want to verify the params match a certain pattern, we should do that explicitly instead (like we are now).